Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -1330,10 +1330,16 @@ void NetworkProcess::setAppBoundDomainsForResourceLoadStatistics(PAL::SessionID
 void NetworkProcess::setManagedDomainsForResourceLoadStatistics(PAL::SessionID sessionID, HashSet<WebCore::RegistrableDomain>&& appBoundDomains, CompletionHandler<void()>&& completionHandler)
 {
     if (auto* session = networkSession(sessionID)) {
-        if (auto* resourceLoadStatistics = session->resourceLoadStatistics()) {
-            resourceLoadStatistics->setManagedDomains(WTFMove(appBoundDomains), WTFMove(completionHandler));
-            return;
-        }
+        supplement<WebCookieManager>()->setHTTPCookieAcceptPolicy(sessionID, WebCore::HTTPCookieAcceptPolicy::AlwaysAccept, [session = WeakPtr { *session }, appBoundDomains = WTFMove(appBoundDomains), completionHandler = WTFMove(completionHandler)]() mutable {
```

Comment:
nit (especially because I introduced it): 
```suggestion
        supplement<WebCookieManager>()->setHTTPCookieAcceptPolicy(sessionID, WebCore::HTTPCookieAcceptPolicy::AlwaysAccept, [session = WeakPtr { *session }, managedDomains = WTFMove(managedDomains), completionHandler = WTFMove(completionHandler)]() mutable {
```

Reply:
Oh gosh! I didn't even notice! LOL.

Reply:
Will fix.